src/Button.vala

```diff
@@ -24,19 +24,27 @@ public class PantheonCalculator.Button : Gtk.Button {
 
     public Button (string label, string? description = null) {
         function = label;
-        var lbl = new Gtk.Label (label);
-        lbl.use_markup = true;
-        add (lbl);
+
+        var lbl = new Gtk.Label (label) {
+            use_markup = true
+        };
+
+        child = lbl;
         tooltip_text = description;
     }
 
-    public override void get_preferred_width (out int minimum_width, out int natural_width) {
-        minimum_width = WIDTH;
-        natural_width = WIDTH;
+    construct {
+        width_request = WIDTH;
+        height_request = HEIGHT;
```

Comment:
If we want to force aspect ratio for these buttons we should be packing them into a Gtk.AspectFrame instead

Reply:
Since the window isn't resizable, this doesn't matter